dataland-frontend/tests/e2e/specs/user-authentication/Register.ts

```diff
@@ -6,6 +6,8 @@ describe("As a user I want to be able to register for an account and be able to
   const randomHexPassword = [...passwordBytes].map((x) => x.toString(16).padStart(2, "0")).join("");
 
   it("Checks that registering works", () => {
+    cy.task("setEmail", email);
+    cy.task("setPassword", randomHexPassword);
```

Comment:
this i would like to understand. Can we discuss that?

Reply:
Sure, we can have a call whenever it fits you.

Reply:
thank you